stack.yaml

```diff
@@ -18,7 +18,11 @@ extra-deps:
           - persistent-postgresql
           - persistent-template
           - persistent-qq
-# :(
+    - lti13-0.1.1.0
+    - yesod-auth-lti13-0.1.1.0
+
```

Comment:
I think we need oidc-client-0.5.1.0 here.

Reply:
Very likely. Sorry about that, my stack doesn't work properly so I can't test full builds (that locale bug is still hitting me extremely badly, making it effectively not usable).